Added Course page

MGafitescu opened the pull request:
Created the Course html page and added the necessary classes in the
style.css files. The course page is the template page for all the tasks
in a course.

MGafitescu committed:
Added Course page:

Created the Course html page and added the necessary classes in the
style.css files. The course page is the template page for all the tasks
in a course.

bogsen commented on course.html:
```diff
@@ -0,0 +1,67 @@
+			<a href="#" class="logo">SkIns</a>
+
+			<a href="#" class="active">Learn</a>
+			<a href="#">Tasks</a>
```
Make this active instead of the other page

bogsen commented on style.css:
```diff
@@ -57,55 +57,67 @@ body > .container {
 	box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.25);
 	border-radius: 6px;
-	height: 10rem;
+  padding: 1rem;
```
Indentation not consistent (entire file)

bogsen reviewed (changes requested):
👍🏻

GeoStefan commented on style.css:
```diff
@@ -57,55 +57,67 @@ body > .container {
 	object-fit: cover;
 	max-width: 12rem;
-	max-height: 100%;
+	max-height: 100%;;
```
There are too many ";".

bogsen commented on style.css:
```diff
@@ -188,20 +206,189 @@ h3 {
+/*Display the course section boxes more larger*/
+/*
+@media screen and (min-width: 40rem) {
+		.flex-container {
```
Indentation

MGafitescu committed:
Edited the files

Changed the active page in course.html, changed the links in course.html
and index.html, fixed the indentation in style.css

MGafitescu committed:
Fixed a validation problem.

MGafitescu committed:
Fixing an error

Replaced the obsolete "center" element with CSS.

MGafitescu committed:
Replaced space with tabs in indentation.

MGafitescu committed:
Replaced a missing tab and added back spaces.

MGafitescu committed:
Final changes.
